src/resource/user/user.service.ts

```diff
@@ -55,4 +75,17 @@ export class UserService {
     this.logger.log(`Create password reset token for user with id ${_id}.`);
     return `${process.env.CLIENT_BASE_URL}/reset-password/${resetToken}`;
   }
+
+  async resetPassword(_id: string, _newPassword: string) {
+    return await this.prisma.user.update({
```

Comment:
Maybe we can have a function to just update the user, instead of using `prisma` separately in each function.

Reply:
This has been updated by creating updateUserById service